### PR TITLE
#159397061: Update a booked menu/meal

### PR DESCRIPTION
**Pull request by larrystone**
**What does this PR do?**
- allow user edit an already booked menu rather than placing a new one

**What are the relevant pivotal tracker stories?**
[#159397061](https://www.pivotaltracker.com/story/show/159397061)

**Screenshot?**
screen shot 2018-08-15 at 3 37 19 am
screen shot 2018-08-15 at 3 39 15 am
screen shot 2018-08-15 at 3 39 33 am

### CODE REVIEW

File: client/src/components/Order/ConfirmOrder.jsx [hunk 1]
```diff
@@ -15,77 +15,85 @@ import Meal from './Meal';
 class ConfirmOrder extends Component {
 
   confirmOrder = () => {
-    const { orderMeal, showToast, toggleModal } = this.props;
-    orderMeal(this.props.mealSelected)
+    const { orderMeal, showToast, toggleModal, menuId, updateOrder, isLoading } = this.props;
+    if (menuId) {
+      // Updating an already created order
```
Comment (Charpell) [inline client/src/components/Order/ConfirmOrder.jsx]: Remove all comments

Comment (larrystone) [reply client/src/components/Order/ConfirmOrder.jsx]: Okay, I had to leave this comment here to make it easy for the person maintaining this code to understand what is happening. Should I still go on to remove it?

Comment (Charpell) [reply client/src/components/Order/ConfirmOrder.jsx]: @CharlesJonah over to you

**Comment by Charpell** [comment]
LGTM

**Pull request merged**